### PR TITLE
Issue #3959: Split and Organize Checkstyle inputs by Test for FileTabCharacter

### PR DESCRIPTION
**Pull request by Kietzmann**
This pull request covers to issue #3959 
What was modified:

Modified paths to test classes
Moved all necessary inputs into /whitespace/filltabcharacter package
Results:

All unit tests passed.
the result of mvn clean verify also OK

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/FileTabCharacterCheckTest.java [hunk 1]
```diff
@@ -56,9 +58,9 @@ public void testDefault() throws Exception {
             "19:25: " + getCheckMessage(MSG_FILE_CONTAINS_TAB),
         };
         final File[] files = {
-            new File(getPath("InputSimple.java")),
+            new File(getPath("SimpleFileTabCharacterInput.java")),
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/FileTabCharacterCheckTest.java]: Input files should be named after their tests.
`FileTabCharacterCheckTest`'s inputs should be named `InputFileTabCharacter...`.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/filetabcharacter/InputFileTabCharacterSimple.java [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+
+/**
+ * Contains simple mistakes:
+ * - Long lines
+ * - Tabs
+ * - Format of variables and parameters
+ * - Order of modifiers
+ * @author Oliver Burn
+ **/
+final class SimpleFileTabCharacterInput
```
Comment (rnveach) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/filetabcharacter/InputFileTabCharacterSimple.java]: input classes must match file name.

Comment (Kietzmann) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/filetabcharacter/InputFileTabCharacterSimple.java]: Recently I made a commit with appropriate changes

**Comment by romani** [comment]
@Kietzmann , please squash all commits in one for final review.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/filetabcharacter/InputFileTabCharacterSimple.java (hunk 2 above)
Comment (rnveach) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/filetabcharacter/InputFileTabCharacterSimple.java]: @Kietzmann This is still not done.

**Comment by Kietzmann** [comment]
@rnveach 
Please make code review.
Thanks.

**Comment by Vladlis** [comment]
@Kietzmann 
Please address @rnveach's point and rename class `SimpleFileTabCharacterInput` to `InputFileTabCharacterSimple`.

Please, write a proper commit message,

**Comment by Kietzmann** [comment]
@Vladlis 
Actually I've hastened with previous commit message. But I've already renamed SimpleFileTabCharacterInput to InputFileTabCharacterSimple.

**Comment by Vladlis** [comment]
@Kietzmann 
You have only renamed file, but class name is still `SimpleFileTabCharacterInput`

**Comment by Kietzmann** [comment]
Ups...
Now it seems that everything correct

**Comment by rnveach** [comment]
@Kietzmann Now you need to rebase on the latest master. When you squashed the commits, somehow you added commits that were not your own so now you have changes outside of `FileTabCharacter`.
We only expect changes to `FileTabCharacterTest` and it's inputs.
Please review your commit before pushing again.

**Comment by Kietzmann** [comment]
@rnveach Seems to be fixed

**Pull request merged**